src/main/java/com/facebook/openwifirrm/ucentral/models/WifiScanEntryResult.java

```diff
@@ -50,8 +48,7 @@ public class WifiScanEntryResult {
 	public String vht_oper;
 	public int capability;
 	public int frequency;
-	/** IE = information element */
-	public JsonArray ies;
+	// ies[] array is not stored directly, but parsed into WifiScanEntry fields
```

Comment:
Just remove the comment here? Or move it to the javadoc of this class.

Reply:
Jeffrey wanted the comment. But I can move it to javadoc